doc/sphinx/conf.py

```diff
@@ -194,7 +194,32 @@ def setup(app):
     'term_pattern',
     'term_pattern_string',
     'command',
-    'symbol' ]]
+    'symbol',
+
+    # added for Ltac chapter
+    'tacarg',
+    'ssr_first_else',
+    'ssrseqarg',
+    'int_or_var',
+    'string_opt',
+    'ssrmmod',
+    'ssrdotac',
+    'ssrclauses',
+    'ssrortacarg',
+    'ssrdgens',
+    'query_command',
+    'int_or_var_opt',
+    'simple_tactic',
+    'reference',
+    'ssrintros_ne',
+    'STRING',
```

Comment:
Yeah, consistency is good.  `STRING` is the token, `string` is a non-terminal.  We should eventually define lexer tokens as productions--useful info and it will naturally link from the rest of the grammar.

```

string: [
| STRING
]
```